data/huion-new-1060-plus.tablet

```diff
@@ -1,6 +1,6 @@
 # HUION
 # New 1060 Plus
-#
+# https://github.com/linuxwacom/wacom-hid-descriptors/files/8801571/sysinfo.H1060P.tar.gz
```

Comment:
please link to the PR that added the file, not the file directly. this ensures that any discussion ideally ends up in one place. Thanks!